bin/jib

```diff
@@ -228,12 +228,11 @@ checkStorage() {
 # @option --gas-price Override the gas price to use for the transaction.
 grantPermissions() {
   ADDRESS_ARRAY=$(awk 'BEGIN { printf "[" } { if (NR == 1) printf "%s", $0; else printf ",%s", $0; } END { printf "]" }' $argc_file)
-  PRIVATE_KEY=$argc_private_key forge script \
-    script/Util.s.sol:GrantPermissions \
+  forge script script/Util.s.sol:GrantPermissions \
     $argc_contract $ADDRESS_ARRAY \
     --sig "run(address,address[])" \
     --rpc-url $argc_rpc_url --broadcast -vvvv \
-    --gas-price "${argc_gas_price:-0}"
```

Comment:
Is this removal meant for this PR?

Reply:
I was doing some drive-by cleanup... that was a shim to compensate for gas prices on Polygon being really high and us having transactions fail repeatedly. I would be fine with reverting that here if you think it's inappropriate.

Reply:
No, hopefully it's not necessary in general. If we did need to add it back though, I would rather make it a global option for all scripts than just on this `grant-permissions` script.